Automattic/jetpack
# My Jetpack: Add a Connections Section component wrapping the Connection Status Card

Adds a Connections Section component wrapping the Connection Status Card to My Jet

Fixes nothing

#### Changes proposed in this Pull Request:

* Wraps ConnectionStatusCard usage into a new component



#### Does this pull request change what data or activity we track or use?
none
#### Testing instructions:

* Checkout this branch
* Build with `jetpack build packages/my-jetpack`.
* Have Jetpack Backup and a plan for it (either Upgrade to Security or Backup)
* Checkout the My Jetpack page.
* Confirm the Connection Card looks the same 🤷
